### DISCUSSION TITLE
How many port does each router have

### DISCUSSION THREAD

**Discussion by orangepineapple** [ANLAB-KAIST/KENSv3#255] [Project Q&A 3. Internet Layer (IP/Routing)]
From my understanding we can use ipv4_t routerIP = *getIPAddr(); to get the IP of whichever router is connected to a given port and we can get the linkcost of that router via the LinkCost() function in the same way. But how many ports does each router have? How do we know when we are done looping through ports? Are neighbors connected to ports in sequential order?

**Comment by seo-rii** [comment]
https://github.com/ANLAB-KAIST/KENSv3/discussions/242
